img2tiles.py

```diff
@@ -1,14 +1,14 @@
+from email.mime import base
```

Comment:
Can you explain why `email.mime.base` is imported?

Reply:
Lol, no idea at what point it ended up in my code. I'll fix it now, thanks for noticing